protostar/starknet_gateway/block_explorer/fake_block_explorer.py

```diff
@@ -0,0 +1,18 @@
+from .block_explorer import (
+    URL,
+    BlockExplorer,
+    ClassHash,
+    ContractAddress,
+    TransactionHash,
+)
+
+
+class FakeBlockExplorer(BlockExplorer):
+    def create_link_to_transaction(self, tx_hash: TransactionHash) -> URL:
+        return ""
+
+    def create_link_to_contract(self, contract_address: ContractAddress) -> URL:
+        return ""
+
+    def create_link_to_class(self, class_hash: ClassHash) -> URL:
+        return ""
```

Comment:
I think we can avoid this class?

Why not assume voyager as default block explorer even in tests? Simply avoid testing block explorer output in assertions.

Reply:
discussed this privately